test/functional/cmdline_options_testresources/src_90/com/ibm/j9/getcallerclass/ReflectionMHTests.java

```diff
@@ -62,7 +62,7 @@ public static boolean test_getCallerClass_Helper_Reflection_fromBootExtWithAnnot
 			cls = (Class<?>) method.invoke(null, new Object[0]);
 
 			boolean isClassNameExpected;
-			if (VersionCheck.major() >= 18) {
+			if ((VersionCheck.major() >= 23) || ((VersionCheck.major() >= 18) && Boolean.getBoolean("jdk.reflect.useDirectMethodHandle"))) {
```

Comment:
Reason I had JDK23 is because that is the version from which the property will not be available. This one works as well. Made changes in https://github.com/eclipse-openj9/openj9/compare/57a5ddfa22038e022ce188f6c7e5374099639523..25c26bfcc6f46952e991489d5cd72a6c8c0001c7

Reply:
It's not in jdk22 either (except for a test); see https://github.com/search?q=repo%3Aibmruntimes%2Fopenj9-openjdk-jdk22%20jdk.reflect.useDirectMethodHandle&type=code.